src/oedisi/types/data_types.py

```diff
@@ -79,15 +79,15 @@ class MeasurementArray(BaseModel):
     Extended by classes:
         "BusArray",
         "EquipmentArray",
-        "EquipmentNodeArray"
+        "EquipmentNodeArray".
     """
 
-    values: List[float]
-    ids: List[str]
+    values: list[float]
```

Comment:
Likewise, ruff here is using lowercase lists and replacing Optional. Not sure if this is what we want or not?

Reply:
See above